packages/calcite-components/src/components/interfaces.ts

```diff
@@ -28,7 +29,7 @@ export type SelectionMode =
   | "multiple";
 export type Scale = "s" | "m" | "l";
 export type Status = "invalid" | "valid" | "idle";
-export type Width = "auto" | "half" | "full";
+export type Width = "s" | "m" | "l" | "auto" | "half" | "full";
```

Comment:
Could you do `export type Width = Scale & "auto" | "half" | "full";`?  Same for `Height` above.

Reply:
On the second thought, I feel like it might make things a bit confusing to read. `Height`, `Width`, and `Scale` are different props even though they have an overlap in type (with type Scale). s/m/l for scale does something else for width/height. Should they just have their own type names? It hurts my eyes to read type Height = Scale :)